test/unit_Polygon.jl

```diff
@@ -107,3 +107,23 @@ for N in [Float64, Float32] # TODO Rational{Int}
     @test ⊆(HPolygon{N}(), p1)
     @test ⊆(p1, BallInf(N[1., 0.], N(1.)))
 end
+
+# Test comparison of directions <= for rational entries
+N = Rational{Int}
+v1 = N[1.0, 0.0]
+v2 = N[1.0, 1.0]
+v3 = N[0.0, 1.0]
+v4 = N[-1.0, 1.0]
+v5 = N[-1.0, 0.0]
+v6 = N[-1.0, -1.0]
+v7 = N[0, -1.0]
+v8 = N[1.0, -1.0]
+v = [v1, v2, v3, v4, v5, v6, v7, v8]
+
+for (i, vi) in enumerate(v)
+    for j in 1:8
+        if i >= j
```

Comment:
simpler: `for j in 1:i`

Reply:
ok let's merge them (though i have to swap the comparison)

Reply:
really?